packages/cascara/src/lib/customPropTypes/json.js

```diff
@@ -1,13 +1,14 @@
 // This propType function should validate if we are using JSON data
 
-const json = (props, propName, componentName) => {
+// eslint-disable-next-line consistent-return -- this can't return anything
```

Comment:
Should be an arrow func.

We don't need to disable the eslint rule here. Can't we just `return JSON.parse(props[propName]);` on line 6 which would resolve the lint error?

Reply:
changed, thanks.